LLMonFHIR/Home.swift

```diff
@@ -12,6 +12,7 @@ struct HomeView: View {
     @State private var showSettings = false
     @State private var showMultipleResourcesChat = false
     @AppStorage(StorageKeys.enableTextToSpeech) private var textToSpeech = StorageKeys.currentEnableTextToSpeech
```

Comment:
We do never control the binding here anywhere so it could all just live in the `@Observable` view model a few levels down, this can be removed here as far as I can see and would also make all the initialization of the subsequent views easier.